api/desecapi/views.py

```diff
@@ -181,21 +202,23 @@ def send_dyn_dns_email(domain):
                                  [self.request.user.email])
             email.send()
 
-        if obj.name.endswith('.dedyn.io'):
-            send_dyn_dns_email(obj)
+        if domain.name.endswith('.dedyn.io'):
```

Comment:
remove static dependency on `'dedyn.io'`

Reply:
nevermind, everything in this function depends on "dedyn.io", and is going to be replaced with the new user management anyways